homeassistant/components/here_travel_time/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "here_travel_time",
   "name": "HERE Travel Time",
   "documentation": "https://www.home-assistant.io/integrations/here_travel_time",
-  "requirements": ["herepy==2.0.0"],
+  "requirements": ["herepy==3.0.2"],
```

Comment:
unrelated commit

Reply:
This is related. We can only have one version of the herepy dependency.
As `here_travel_time` and `here_weather` are using the same dependency I had to update them.

`here_travel_time` does not mock the lib for tests but mocks the api response. This is why I had to adjust the fixture for the new lib version as well